starter/angular-xeditable/js/xeditable.js

```diff
@@ -857,7 +857,7 @@ function($parse, $compile, editableThemes, $rootScope, $document, editableContro
         // element wrapped by form
         if(ctrl[1]) {
           eFormCtrl = ctrl[1];
-          hasForm = true;
+          hasForm = attrs.eSingle === undefined;
```

Comment:
This is already upstream master.  Why is it in this pull req? https://github.com/vitalets/angular-xeditable/blob/master/starter/angular-xeditable/js/xeditable.js#L872